azext_edge/tests/edge/support/create_bundle_int/test_arccontainerstorage_int.py

```diff
@@ -59,8 +62,55 @@ def test_create_bundle_arccontainerstorage(init_setup, tracked_files):
         )
 
     check_workload_resource_files(
-        file_objs=file_map,
+        file_objs=acs_file_map,
         expected_workload_types=expected_workload_types,
         prefixes=workload_resource_prefixes,
         bundle_path=bundle_path,
     )
+
+    # validate azure-arc-acstor if exists
+
+    if "acstor" not in file_map:
+        return
+
+    acstor_file_map = file_map["acstor"]
+
+    expected_workload_types = ["daemonset", "deployment", "pod", "replicaset", "service", "configmap"]
+    expected_types = set(expected_workload_types).union(CONTAINERSTORAGE_API_V1.kinds)
+    assert set(acstor_file_map.keys()).issubset(set(expected_types))
+
+    workload_resource_prefixes = [
+        "acstor-action",
+        "acstor-agent",
+        "acstor-api-rest",
+        "acstor-capacity",
+        "acstor-cert-manager",
+        "acstor-crd",
+        "acstor-csi",
+        "acstor-etcd",
+        "acstor-io",
+        "acstor-ndm",
+        "acstor-operator",
+        "acstor-prereq",
+        "acstor-scripts",
+        "acstor-support-bundle",
```

Comment:
at this point - can you just use acstor? or will something else that we don't want get captured?

Reply:
do we not want to test for more specific prefixes?

Reply:
this size of prefixes to check is getting a bit big - so if possible, why simplify/shorten the list

Reply:
updated